src/libraries/Directory.Build.props

```diff
@@ -329,4 +329,9 @@
     <CLSCompliant Condition="'$(CLSCompliant)' == ''">true</CLSCompliant>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <ILLinkTrimAssemblyArtifactsRootDir>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)</ILLinkTrimAssemblyArtifactsRootDir>
```

Comment:
Use NormalizeDirectory?

Reply:
In general, "directory" properties in MSBuild need to contain a trailing slash. It is standard practice.

Reply:
Thanks. Done